fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/api/content/registry/v1/FlattenableBlockRegistry.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.content.registry.v1;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+
+import net.fabricmc.fabric.api.content.registry.v1.util.Taggable2ObjectMap;
+import net.fabricmc.fabric.impl.content.registry.ToolBlockTransformationRegistries;
+
+/**
+ * Registry of Blocks that when flattened turn into a specific BlockState.
```

Comment:
What do you *do* to flatten something? Is it crop trampling or converting grass to paths or tilling?

Reply:
Flatten it with a shovel, such as a grass path. This is the term mojang uses in subtitles/lang for the action